services/_shared/personas/code_reviewer/dispatch.py

```
@@ -637,9 +646,45 @@ def _publish_shape(
     return evaluation.conclusion, "COMMENT"
 
 
+def _precedent_notes_for(
+    repo_full: str, findings: "tuple[Finding, ...] | list[Finding]",
```

Ruff (0.15.20)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)